packages/web-pkg/src/components/TextEditor/TextEditor.vue

```diff
@@ -177,6 +189,11 @@ export default defineComponent({
     @apply break-keep;
   }
 
+  .md-editor-preview > ol,
+  ul {
```

Comment:
I don't think we want this globally on `ul`, no? At least it gives me a weird thing here:

<img width="88" height="45" alt="Image" src="https://github.com/user-attachments/assets/2f9c1b53-0a00-4458-9743-b6bbe070f031" />